package-testing/php-sdk-relay/src/AssignmentHandler.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Eppo\SDKTest;
+
+use Eppo\EppoClient;
+use Eppo\Exception\EppoClientException;
+use Exception;
+
+class AssignmentHandler
+{
+    private static array $methods = [
+        'INTEGER' => 'getIntegerAssignment',
+        'STRING' => 'getStringAssignment',
+        'BOOLEAN' => 'getBooleanAssignment',
+        'NUMERIC' => 'getNumericAssignment',
+        'JSON' => 'getJSONAssignment',
+    ];
+
+    public function __construct(private EppoClient $eppoClient, private TestLogger $assignmentLogger)
+    {
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function getAssignment(array $payload): array
+    {
+        $variationType = $payload['variationType'];
```

Comment:
It _may_ be too late in the game to change this, but "variation type" is not a good descriptor for this. Variation Type already means the type of value variations hold (e.g., booleans, strings, integer, numeric JSON) and now were overloading it to also mean the type of assignment we are requesting (note: bandits return string variation types... for now!). 

I think a better name for this would be _assignment_ type.

Sorry I didn't think of this until now

Reply:
👍   Let's plumb it through the test cluster as `assignmentType` - it's not too late.
The presence of "variationType" is how the test runner decides between the `/flags` and the `/bandits` endpoints of the relay server (should probably use `actions` instead). I had considered specifying an endpoint per assignment type and leave this field out altogether, but it makes for a lot of duplicated code.